demo/checkbox-theme-variants-demos.html

```diff
@@ -24,6 +24,25 @@ <h4>Vertical</h4>
       </template>
     </vaadin-demo-snippet>
 
+    <h3>Helper text position</h3>
+    <vaadin-demo-snippet id="text-field-theme-demos-sizes">
+      <template preserve-content>
+        <h4>Below</h4>
+        <vaadin-checkbox-group label="Label" helper-text="Helper text below text field">
```

Comment:
Mentions "text field". Maybe just "...below the field"?

Reply:
Done.